08-communicating-results.Rmd

```diff
@@ -117,7 +117,7 @@ R has many packages for creating compelling and insightful charts. We will focus
 
 {ggplot2} follows the "grammar of graphics," a framework that incrementally adds layers of chart components. We can customize visual elements such as scales, colors, labels, and annotations to enhance the understanding of data.
 
-Let's walk through an example using the ANES 2020 data. As usual, we begin by setting up our survey object:
+Let's walk through an example using the ANES 2020 data. As usual, we begin by setting up our survey object: 
```

Comment:
All of this code we already walk them through in chapter 4.  let's create a setup section at the beginning of the chapter and just read in the data like we do in chapters 5-7.